File: Temp/leetcode/editor/cn/[258]各位相加.py
# 258 各位相加
# https://leetcode.cn/problems/add-digits/


# leetcode submit region begin(Prohibit modification and deletion)
class Solution:
    def addDigits(self, num: int) -> int:
        while num >= 10:
            s = 0
            while num:
                s += num % 10
                num //= 10
            num = s
        return num
# leetcode submit region end(Prohibit modification and deletion)

File: Temp/leetcode/editor/cn/[258]各位相加.c
// 258 各位相加
// https://leetcode.cn/problems/add-digits/


//leetcode submit region begin(Prohibit modification and deletion)
int addDigits(int num) {
    while (num >= 10) {
        int s = 0;
        while (num) {
            s += num % 10;
            num /= 10;
        }
        num = s;
    }
    return num;
}
//leetcode submit region end(Prohibit modification and deletion)
